packages/ionic/src/lib/integrations/capacitor/index.ts

```diff
@@ -24,13 +26,22 @@ export class Integration extends BaseIntegration {
       if (details.enableArgs[1]) {
         packageId = details.enableArgs[1];
       }
+
+      /*
+       * Use parseArgs on the process variables because integrations enable has its own set of options outside
+       * of the options that we will pass to capacitor
+       */
+      const parsedArgs = parseArgs(process.argv.slice(4));
```

Comment:
Why not parse `details.enableArgs`?

Reply:
Because the options are not available since it is not defined as an actual option to the command.

Reply:
Ok, updated based on recommendation